index.bs

```diff
@@ -427,6 +427,55 @@ spec:reporting; urlPrefix: https://w3c.github.io/reporting/
       </div>
     </section>
   </section>
+  <section>
+    <h3 id="meta-element">
+      The `<meta>` element
+    </h3>
+
+    A {{Document}} may deliver a policy via one or more HTML <{meta}> elements
+    whose <{meta/http-equiv}> attributes are an <a>ASCII case-insensitive</a>
+    match for the string "`Feature-Policy`". For example:
+
+    <div class="example">
+      <pre highlight="html">
+        &lt;meta http-equiv="Feature-Policy" content="fullscreen 'none'; geolocation 'none'"&gt;
+      </pre>
+    </div>
+
+    Implementation details can be found in HTML's <a>Feature Policy state</a>
+    `http-equiv` processing instructions [[!HTML]].
+
+    Authors are <em>strongly encouraged</em> to place <{meta}> elements as early
```

Comment:
Is it important that the `<meta charset>` comes before `<meta http-equiv>`s? I asked the same thing in https://github.com/w3c/webappsec-csp/issues/387.

Reply:
Good question, and thanks for the heads up. I'll plan to monitor the conversation there and copy any changes over.